### ISSUE TITLE
post/apache-kafka-ccdak-exam-notes/

### ISSUE THREAD

**Issue by ashishlahoti** [ashishlahoti/comments-codingnconcepts.com#2]

**Comment by ashishlahoti** [comment]
Feel free to comment on the post for any query regarding **CCDAK**

**Comment by AashishKotagiri** [comment]
very well written article!! really glad I found it, its helping me a lot in the preparation.

**Comment by erjan** [comment]
what do u mean by 'there are many other kafka certifications'? there is only 1 - this is from confluent. no other vendor is popular or has established certifications

**Comment by ashishlahoti** [comment]
@erjan You have answered your question :) there are other vendors who claims that they provide Kafka certification but they are not as popular or as established as **confluent**

**Comment by erjan** [comment]
"Message Key
Producer can choose to send a key with message."

maybe typo but it should be "producer can choose to send msg with key" ?

thx u for the great guide , i m prepping for the ccdak now!

**Comment by ashishlahoti** [comment]
@erjan I have updated the statement in "Message Key" section. Feel free to comment if you see any other mistakes or typos. Thanks.

**Comment by BananaEgg** [comment]
Both on Apache's and Confluent's documentation they use bootstrap-server to create a new topic.
I seem to remember reading that the both work, but creating with zookeeper is deprecated.
what would be the correct answer in the exam?

**Comment by ashishlahoti** [comment]
@BananaEgg You are right. Zookeeper is deprecated in the latest version of Kafka and it will be removed soon in upcoming versions. You should use `bootstrap-server` to create new topic. I have updated the **Kafka CLI** section of this post.

Correct answer is `bootstrap-server`.

**Comment by Sophiamiaa98** [comment]
I Prepared my Confluent Certified Developer for Apache Kafka CCDAK Exam within a week with the help of Validexamdumps Updated Certified Developer for Apache Kafka CCDAK Practice Test Material. On the Final Exam Day, I Easily Attempted All questions and I Got Success in Confluent CCDAK exam the First Attempt.
Download Full Practice Exam Material Here: https://www.validexamdumps.com/confluent/ccdak-dumps

**Comment by aditya81070** [comment]
@ashishlahoti, Does the certifications ask questions for latest versions of Kafka or they ask from a specific version? 

Also, I have only seen 2-3 questions related to Kafka monitoring as they are more aligned with admin certification of Kafka. So should I read about the monitoring metrics as well?

**Comment by ashishlahoti** [comment]
@aditya81070 Hey Aditya, Certification asks questions for the latest version of Kafka.

These exam notes are for CCDAK (Confluent Certified Developer for Apache Kafka®). These are useful for you but yes, you need additional learning including monitoring metrics for CCAAK (Confluent Certified Administrator for Apache Kafka®)

**Comment by dabwali** [comment]
do i need to study KAFKA CONNECT as well for the exam?

**Comment by gulpinDev** [comment]
are these notes still valid for 2023 CCDAK exams? great job, btw :)

**Comment by OgradyJ** [comment]
These notes are still great. Other than the above sample questions I'd recommend the following resource for more practice questions: https://www.examskit.com/confluent/ccdak/pdf I found their premium questions very helpful during my exam preparation.

**Comment by BIPINAIREE** [comment]
Hello ashishlahoti . i am Preparing for CCAAK(Confluent certified administrator for apache kafka). Is there any learning material like CCDAK for CCAAK.Will this CCDAK will helpful for CCAAK?

**Comment by jimmytheSer** [comment]
I highly recommend https://www.itexamspro.com/confluent/ccdak-dumps.html to anyone looking to pass their CCDAK exam. Their exam dumps and practice tests helped me to achieve success with ease.
